framework/Steps/IOStep.py

```diff
@@ -121,7 +121,12 @@ def _localInitializeStep(self,inDictionary):
       elif isinstance(inDictionary['Input'][i], (Models.ROM, Models.ExternalModel)):
         # ... file
         if isinstance(outputs[i],Files.File):
-          self.actionType.append('MODEL-FILES')
+          #XXX switching on the file extension is not very
+          # extendable.  Possibly should have a type or something
+          if 'py' == outputs[i].getExt().lower():
+            self.actionType.append('MODEL-PYOMO')
+          else:
+            self.actionType.append('MODEL-FILES')
```

Comment:
Yeah, this was initially Josh's and Andrea's scapegoat to trigger pyomo print in ROM.py. Since it did work, I did not see the necessity to rewrite this method for more elegant forms. Is there any suggestions?

Reply:
Let's have a discussion on Monday meeting

Reply:
We have since updated this to use the file type.